week-2-wide-perspective/w02e03-pagination-problem/src/app.js

```diff
@@ -1,12 +1,15 @@
 import express from 'express'
 import { fruitsController } from './fruits/fruits.controller.js'
 import { mainController } from './main/main.controller.js'
+import { paginate } from './middlewares/paginate.middleware.js'
 import { usersController } from './users/users.controller.js'
 
 export const app = express()
 
 app.use(express.json())
 
+app.use(paginate)
```

Comment:
do poprawki tutaj - bo nie miało być to wczepione globalnie,
tylko te ścieżki w `routes` które faktycznie potrzebują paginacji powinny mieć wpięte middleware.